.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: objective-c
+before_install:
+    - (ruby --version)
```

Comment:
Ah, gotcha. Thought it might trigger some action related to the ruby version installed. :+1: